Form/Type/DynamicFormType.php

```diff
@@ -149,6 +157,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'mapped' => false,
         ]);
 
+        if ($this->honeyPotField) {
+            $builder->add(
+                str_replace(' ', '_', strtolower($this->honeyPotField)),
+                EmailType::class,
+                [
+                    'label' => $this->honeyPotField,
+                    'mapped' => false,
+                    'block_prefix' => 'honeypot',
```

Comment:
should also be `required` => false

Reply:
right! I just want to add this right now, but I saw you already did it :-).